utils/docker/Dockerfile.leap.15

```diff
@@ -122,11 +79,12 @@ ARG DAOS_TARGET_TYPE=release
 # Now do an update to ensure software is up to date for the deps build.  If the
 # src hasn't changed then this won't do anything, but if it has then we want to
 # ensure that latest dependencies are used.
+# The dnf upgrade can add or re-enable distro repositories.
 USER root:root
-RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || {                                                  \
-        dnf upgrade                                                                       \
-            --exclude=fuse,fuse-libs,fuse-devel,libraft0,raft-devel,mercury,mercury-devel \
-        dnf clean all;                                                                    \
+RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || {                                                     \
+        dnf upgrade                                                                          \
+            --exclude=fuse,fuse-libs,fuse-devel,libraft0,raft-devel,mercury,mercury-devel && \
```

Comment:
This looks like an interesting bug-fix, what was the code before doing?

Reply:
It was probably attempting to install "dnf", "clean", and "all" packages instead of running it as a command.